src/fsharp/TastOps.fs

```diff
@@ -635,6 +674,20 @@ let rec stripTyEqnsA g canShortcut ty =
 
 let stripTyEqns g ty = stripTyEqnsA g false ty
 
+let rec stripTupInfoEqns aexpr = 
+    match aexpr with 
+    //| TupInfo.Var v -> 
```

Comment:
Similar to previous comment elsewhere. Just wondering why some things are commented out?